### PR TITLE
CHE-4211: Fix InnerClassAndLambdaDebuggingTest test

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
* Fixes java debugger server side part concerning adding breakpoints into local classes
* Removes checking of unexisted functionality of the eclipse che debugger

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/4211

### CODE REVIEW

**Review by vparfonov — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java [hunk 1]
```diff
@@ -140,7 +140,6 @@ public void shouldDebugMethodLocalInnerClass() {
 
     // then
     editor.waitActiveBreakpoint(53);
-    debugPanel.waitTextInVariablesPanel("methodValue=\"App method local inner test\"");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java:143]: Could you, please, explain, why did you remove this check? Does it mean that there will be no values inside the _Variables Panel_?

Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java:143]: Why did you decide that no variables will be inside the Variable Panel?
It means that `methodValue` is unavailable only.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java:143]: But **methodValue** variable should be there in _Variable Panel_ so as it is accessible from inside the **MethodLocalInner::getValue()** method and is returning by this method.
```javascript
    public String getAppValue() {
        final String methodValue = "App method local inner test";

        class MethodLocalInner {
            String getValue() {
line 53:      return methodValue;
            }
        }

        return new MethodLocalInner().getValue();
    }
```

Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java:143]: Sure, but the reference Eclipse IDE doesn't show that variable too. I think that intelejii does some trick to show that.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java:143]: You are right :-(
screenshot from 2017-11-17 16-57-34

Tested on Eclipse IDE for Java Developers, Version: Neon Release (4.6.0)
